### DISCUSSION TITLE
Kitchen sink

### DISCUSSION THREAD

**Discussion by MatteoJoliveau** [oxidrive/oxidrive#33] [Ideas]
Let's dump all our random ideas here for future features. Interesting ideas will be moved to their own discussion thread for exploration.

**Comment by fedeizzo** [comment]
- [ ] Share a resource for X days using this links.
- [ ] Share a resource until <date> in write/read mode, after <date> convert into read mode.
- [ ] Share a directory in write mode (no read mode).
